packages/graphiql-toolkit/src/storage/custom.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * This function enables a custom namespace for localStorage
+ */
+
+import { Storage } from './base';
+
+export function createLocalStorage(namespace: string): Storage {
+  const storage = {
```

Comment:
```suggestion
  // you can re-use this key in the methods below
  const storageKey = `${namespace}:${key}`;
  const storage = {
```

Reply:
Yes, that's the part definely shoule be improved! Thanks @acao